config/main.py

```diff
@@ -1754,17 +1766,18 @@ def add_portchannel(ctx, portchannel_name, min_links, fallback):
         fvs['min_links'] = str(min_links)
     if fallback != 'false':
         fvs['fallback'] = 'true'
-    db.set_entry('PORTCHANNEL', portchannel_name, fvs)
+    yvs = YangValidationService()
+    if not yvs.validate_set_entry('PORTCHANNEL', portchannel_name, fvs):
+        ctx.fail("Invalid configuration based on PortChannel YANG model")
+    else:
+        db.set_entry('PORTCHANNEL', portchannel_name, fvs)
+
```

Comment:
It seems we always validate yang no matter `yang_config_validation` is enabled or not?
Maybe we are supposed to validate it based on the `yang_config_validation`?